src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java

```diff
@@ -48,4 +47,12 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         return response;
     }
+
+    protected void updateCaseStatus(SscsCaseData caseData) {
+        PostHearing postHearing = caseData.getPostHearing();
+        if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
```

Comment:
NPE if setAside is null

Reply:
I think the best approach for these methods would be a switch statement for PostHearingType enum.

Reply:
As above, this will require a refactor for the next Post Hearing type. Can be passed on to the future ticket.